src/HH_BTag.cpp

```diff
@@ -0,0 +1,62 @@
+#include "../interface/HH_BTag.h"
+
+namespace hh_btag{
+
+HH_BTag::HH_BTag(const std::array<std::string, HH_BTag::n_models>& models)
+{
+    for(size_t n = 0; n < HH_BTag::n_models; ++n) {
+        nn_descs.at(n).graph.reset(tensorflow::loadGraphDef(models.at(n)));
+        nn_descs.at(n).session = tensorflow::createSession(nn_descs.at(n).graph.get());
+        nn_descs.at(n).input_layer = nn_descs.at(n).graph->node(0).name();
+        nn_descs.at(n).output_layer = nn_descs.at(n).graph->node(nn_descs.at(n).graph->node_size() - 1).name();
+    }
+
+}
+
+std::vector<float> HH_BTag::GetScore(const std::vector<float>& jet_pt, const std::vector<float>& jet_eta,
+                                     const std::vector<float>& rel_jet_M_pt, const std::vector<float>& rel_jet_E_pt,
+                                     const std::vector<float>& jet_htt_deta, const std::vector<float>& jet_deepFlavour,
+                                     const std::vector<float>& jet_htt_dphi, int sample_year, int channelId, float htt_pt,
+                                     float htt_eta, float htt_met_dphi, float rel_met_pt_htt_pt,
+                                     float htt_scalar_pt, int parity)
+{
+    tensorflow::Tensor x(tensorflow::DT_FLOAT, tensorflow::TensorShape{1, 10, 15});
```

Comment:
- `static constexpr max_n_jets = 10;` => use it here and below instead of 10.
- `static constexpr n_variables = 15;` => use it here.